docs/submission/clinical-data-validation-rules.md

```diff
@@ -6,8 +6,52 @@ platform_key: DOCS_CLINICAL_VALIDATION_RULES
 
 ![Reminder Banner](/assets/submission/banner-reminder.svg)
 
+## Clinical Data Encoding Rules
+
+### Identifier fields
+
+The data dictionary contains certain data elements regarded as "identifiers". These fields have an ![ID](/assets/submission/dictionary-id.svg) descriptor in the data dictionary and include:
+
+- Donor: `submitter_donor_id`
+- Specimen: `submitter_specimen_id`
+- Sample: `submitter_sample_id` 
+- Primary Diagnosis: `submitter_primary_diagnosis_id`
+- Treatment: `submitter_treatment_id`
+- Follow Up: `submitter_follow_up_id`
+
+These fields must to be coded specifically for ICGC-ARGO purposes using the following rules:
+
+- These identifiers should not be derived from IDs such as biobank or hospital identifiers or any other personal identifying information. These IDs are to be coded in such a way that they cannot be tracked back to the individual donors, except by the submitting program. Only the program will keep the key that will permit the data to be linked back to the individual donors. This key must not be communicated to the data users.
+- Identifiers are assigned by each submitting program and must be unique within all the data submitted by that program (no duplicate IDs allowed).
+- These identifiers should be values that uniquely identify the entity they refer to.
+- Identifiers referring to the same entity should be consistent across different submissions from the submitting program, and should not be re-used to refer to two different entities. For example, the same donor should not be assigned different identifiers in different files or subsequent data submissions.
+- Identifiers cannot begin with DO, SP, SA, PD, TR or FU. These prefixes are reserved for ICGC-ARGO IDs.
+
+### Primary Diagnosis, Treatment and Follow Up Identifiers:
+
+These identifier fields allow linking of the different clinical events and should be coded using the following rules:
+
+- Each primary diagnosis should be assigned a unique `submitter_primary_diagnosis_id`. In the case of multiple primary diagnoses, each primary diagnosis should be a different `submitter_primary_diagnosis_id`. You will be required to submit the `submitter_primary_diagnosis_id` in the `Specimen` file - this provides information about which primary diagnosis the specimen is linked to. The `submitter_primary_diagnosis_id` is also required in the `Treatment` table, so it is understood which primary diagnosis the treatment is being administered for.
```

Comment:
Each primary diagnosis should be assigned a unique `submitter_primary_diagnosis_id`. In the case of multiple primary diagnoses, each primary diagnosis should be a different `submitter_primary_diagnosis_id`. 

These two sentences seem to say the same thing? could change to:
Each primary diagnosis should be assigned a unique `submitter_primary_diagnosis_id`, so in the case where a donor has multiple primary diagnoses, each primary diagnosis should have a different `submitter_primary_diagnosis_id`.